rsts/concepts/basics.rst

```diff
@@ -96,10 +96,10 @@ Concepts
 
     .. link-button:: divedeep-data-management
         :type: ref
-        :text: How Flyte Manages Data?
+        :text: How Flyte Manages Data
         :classes: btn-block stretched-link
     ^^^^^^^^^^^^
-    What is **metadata**? How large amounts of **raw data** is handled? How data flows between tasks?
+    What is **metadata**? How are large amounts of **raw data** handled? How does data flow between tasks?
```

Comment:
```suggestion
    As a data-aware orchestration tool, Flyte only keeps track of the **metadata** passing through your tasks and workflows, keeping **raw data** within your own data stores.
```

Reply:
Thanks Niels. Will run the editorial review after earlier conflicts have been resolved.